frontend/src/slices/orders/selectors.ts

```diff
@@ -1,5 +1,24 @@
-import { type RootState } from '~/libs/types/types.js';
+import { type DataStatus } from '~/libs/enums/data-status.enum.js';
+import {
+  type OrderResponseDto,
+  type RootState,
+  type ValueOf,
+} from '~/libs/types/types.js';
+
+import { type RouteData } from './libs/types/types.js';
 
 const selectPrice = (state: RootState): number => state.orders.price;
 
 export { selectPrice };
+
+const selectOrder = (state: RootState): (OrderResponseDto | undefined)[] => {
```

Comment:
array of undefined ?

Reply:
It was in the slice in the old version, it was removed there and forgotten here.